contracts/HubPool.sol

```diff
@@ -290,17 +307,20 @@ contract HubPool is Testable, Lockable, MultiCaller, Ownable {
         // Decrement the unclaimedPoolRebalanceLeafCount.
         refundRequest.unclaimedPoolRebalanceLeafCount--;
 
+        _sendTokensToChainAndUpdatePooledTokenTrackers(
```

Comment:
both the actions of sending tokens to a given chain and updating that tokens Pooled Token trackers happen in the same method to save on having to iterate over the `l1Tokens` array multiple times (see function implementation lower down in this contract).